### PR TITLE
VP-7323: Fix icon for member controller

### PR DESCRIPTION
**Pull request by Vectorfield4**
Jira: https://virtocommerce.atlassian.net/browse/VP-7323

### CODE REVIEW

File: src/VirtoCommerce.CustomerModule.Web/Scripts/blades/member-list.tpl.html [hunk 1]
```diff
@@ -48,7 +48,7 @@
 <script type="text/ng-template" id="member-list-icon.cell.html">
     <div class="ui-grid-cell-contents">
         <div class="product-img" ng-attr-title="{{COL_FIELD}}">
-            <i class="table-ico" ng-class="row.entity._memberTypeIcon ? row.entity._memberTypeIcon : 'fa-user'"></i>
+            <i class="table-ico" ng-class="row.entity._memberTypeIcon ? row.entity._memberTypeIcon : 'fa fa-user'"></i>
```
Comment (eMazeika) [inline src/VirtoCommerce.CustomerModule.Web/Scripts/blades/member-list.tpl.html:51]: not sure if this is the right place to fix blade header icon

Comment (Vectorfield4) [reply src/VirtoCommerce.CustomerModule.Web/Scripts/blades/member-list.tpl.html:51]: Its look like its proper place
image
image

Comment (Vectorfield4) [reply src/VirtoCommerce.CustomerModule.Web/Scripts/blades/member-list.tpl.html:51]: Yea, already fixed here https://github.com/VirtoCommerce/vc-module-customer/releases/tag/3.22.0

**Comment by Vectorfield4** [comment]
Already fixed in release https://github.com/VirtoCommerce/vc-module-customer/releases/tag/3.22.0

**Pull request closed without merging**